cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/repositories.go

```diff
@@ -152,6 +157,11 @@ func newHelmRepoCrd(repo *HelmRepository, secret *k8scorev1.Secret, imagePullSec
 			Description:           repo.description,
 			PassCredentials:       repo.auth != nil && repo.auth.PassCredentials,
 			Interval:              repo.interval,
+			// TODO(agamez): are more fields here if they're requested
```

Comment:
Is this meant to be:
```suggestion
			// TODO(agamez): add more fields here if they're requested
```